workflow/ichorCNA.wdl

```diff
@@ -0,0 +1,130 @@
+version 1.0
+workflow ichorCNA {
+
+  input {
+
+    String? modules = "ichorcna/0.2 hmmcopy-utils"
+    File input_bam
+    File input_bam_index
+    String sampleName
+    Boolean includeHOMD
+    Int maxCNstates
+    Int window
+    Int qual
+    Int mem
+    String chromosome_wig
+  }
+
+call runReadCounter{
+  input:
+    input_bam=input_bam,
+    input_bam_index=input_bam_index,
+    sampleName=sampleName,
+    window=window,
+    qual=qual,
+    modules=modules,
+    mem=mem,
+    chromosome_wig=chromosome_wig
+ }
+
+call runIchorCNA {
+input:
+  includeHOMD=includeHOMD,
+  maxCNstates=maxCNstates,
+  sampleName=sampleName,
+  modules=modules,
+  mem=mem,
+  bamWig=runReadCounter.bamWig
+
+ }
+}
+
+
+task copyOverBam {
```

Comment:
I am not sure why this is here, is it used anywhere?

Reply:
This task has been removed.